charts/incubator/hyperswitch-helm/templates/db/hyperswitch-db-job.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   name: create-hyperswitch-db-{{ .Values.services.router.version }}
   labels:
     app: create-hyperswitch-db-{{ .Values.services.router.version }}
+  annotations:
```

Comment:
do we need this annotation. helm install will fail if the installation takes more time 
<img width="392" alt="Screenshot 2024-01-18 at 12 25 10 PM" src="https://github.com/juspay/hyperswitch-helm/assets/20727598/7b3b62b5-c38f-4417-8359-a5955e465724">